client/src/fluid/Fluid.ml

```diff
@@ -5417,13 +5417,17 @@ let pasteOverSelection (data : clipboardContents) (astInfo : ASTInfo.t) :
         ->
           (* Paste into a string, to take care of newlines.
            * Note: pasting before an open quote
-           * places the caret one unit left of the text end *)
+           * places the caret at the text end *)
           let replacement =
             E.EString (id, String.insertAt ~insert:text ~index:(offset - 1) str)
           in
           let newAST = FluidAST.replace ~replacement id ast in
           let caretTarget =
-            CT.forARStringOpenQuote id (offset + String.length text)
+            match offset with
```

Comment:
I think an if statement here would be better, since we don't expect to have other offsets.

Reply:
Right, makes sense.